### PR TITLE
JIRA Fixed - CEP -1286 - Wizard preview port offset mode not displaying bug fixed

### PR DESCRIPTION
**Pull request by keizer619**

### CODE REVIEW

File: apps/portal/templates/gadgets/igviz/js/main.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ if(type === "realtime") {
     columns = gadgetConfig.columns;
     //subscribe to websocket
     subscribe(datasource.split(":")[0], datasource.split(":")[1], '10', 'carbon.super',
```
Comment (suhothayan) [inline apps/portal/templates/gadgets/igviz/js/main.js]: why are we using carbon.supper here ?

File: apps/portal/js/wizard.js [hunk 2]
```diff
@@ -93,7 +94,8 @@ $("#previewChart").click(function() {
             } else {
                 //TODO DOn't do this! read this from a config file
                 subscribe(streamId.split(":")[0], streamId.split(":")[1], '10', 'carbon.super',
```
Comment (suhothayan) [inline apps/portal/js/wizard.js]: why are we using carbon.supper here ? does it works for tenant?

Comment (keizer619) [inline apps/portal/js/wizard.js]: @suhothayan 
I checked the code and dashboard, this value is used to created the directory for domain where gadgets are stored. This does not work in tenant mode. I created a JIRA(CEP-1291) for above bug.

File: apps/portal/templates/gadgets/igviz/js/main.js (hunk 1 above)
Comment (keizer619) [inline apps/portal/templates/gadgets/igviz/js/main.js]: @suhothayan 
I checked the code and dashboard, this value is used to created the directory for domain where gadgets are stored. This does not work in tenant mode. I created a JIRA(CEP-1291) for above bug.

**Pull request merged**